src/database/backup/scripts/backup.sh

```
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Database backup script for Revo-Backend
+# Usage: ./backup.sh [label]
+
+# Load environment variables
+if [ -f .env ]; then
+  export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)